### DISCUSSION TITLE
Trackpad...cause i dont use Mouse and Keyboard

### DISCUSSION THREAD

**Discussion by lllDEMON** [moraroy/NonSteamLaunchers-On-Steam-Deck#43] [Ideas]
Is there a way to add Trackpad support to navigate battle.net(not sure if it doesnt work on other apps as well)?

**Comment by moraroy** [comment]
@lllDEMON already working on it!

**Reply by lllDEMON** [reply]
Sweet you're awesome. Been great so far otherwise.

**Comment by moraroy** [comment]
@lllDEMON this has been added as of late.

**Reply by lllDEMON** [reply]
You sir are amazing
